specification/resources/resource-manager/readme.md

```diff
@@ -40,24 +40,20 @@ tag: package-locks-2016-09
 tag: package-policy-2019-06
 ```
 
-``` yaml $(package-policy)
```

Comment:
same as above

Reply:
this was suplicated, and second has overwritten first resulting in wrong version included in some sdks